next/components/forms/segments/AccountNavBar/AccountNavBar.tsx

```diff
@@ -11,10 +10,12 @@ import useAccount, { UserData } from '@utils/useAccount'
 import cx from 'classnames'
 import HamburgerMenu from 'components/forms/segments/HambergerMenu/HamburgerMenu'
 import Button from 'components/forms/simple-components/Button'
+import Menu from 'components/forms/widget-components/Menu/Menu'
```

Comment:
probably not widget-components but .. simple-components ?

the reasoning behind "widget" components is that they should all be used as RJSF widgets

we'll likely be doing a cleanup of components structure when we start merging bratislava.sk more closely with with account page